src/css/details.css

```diff
@@ -0,0 +1,81 @@
+.img-book-details {
+  background-color: #d8d8d8;
+  height: 270px;
+  width: 190px;
+}
+
+.book-info-container {
+  justify-content: space-between;
+  margin-left: 20px;
+}
+
+.book-description {
+  color: #9b9b9b;
+  font-size: 15px;
+  height: auto;
+  min-height: 60px;
+  text-align: justify;
+  width: 560px;
```

Comment:
don't use a big fixed width in pixels! use width: 100%; max-width: 560px; so it does not break in responsive